scripts/name_giver.py

```diff
@@ -0,0 +1,73 @@
+import os
+import argparse
+import uuid
+
+"""
+Give unique names to all image files in a given folder (and its subfolders).
+"""
+
+IMG_EXTENSIONS = ['.jpg', '.JPG', '.jpeg', '.JPEG', '.png', '.PNG']
```

Comment:
i think we have moved constants to a localized place. this shouldnt need to be redefined. ask @ZPBerg i believe he moved them to a central location.